integration_test/third_party_apps_data/applications/mssql/enable

```diff
@@ -1 +1,14 @@
-'Noop; MSSQL monitoring is on by default.'
\ No newline at end of file
+Add-Content 'C:\Program Files\Google\Cloud Operations\Ops Agent\config\config.yaml' "
+metrics:
+  receivers:
+    mssql_v2:
+      type: mssql
+  service:
+    pipelines:
+      mssql_v2:
+        receivers:
+        - mssql_v2
```

Comment:
Unfortunately there's currently a hard limit of testing one config per application under third_party_apps_data/applications, so we can only either get v1 or v2 this way.

For the sake of just increasing test coverage to both versions, I would suggest having a separate folder under third_party_apps_data/applications for v2 of the receiver -- essentially identical to what you've got in the other PR but with the `enable` config tweaked for `v2`. Fortunately the folder name is not coupled to the app name so you should be able to get away with naming the folder `mssql_v2` or something.

This is admittedly not ideal either because it keeps the metadata.yaml files separate, so separate generated docs, separate expected logs, separate descriptions, etc. We might have to live with that for now until we revisit the integration tests.

CC @sophieyfang @martijnvans do you have any thoughts on how to organize this? This PR introduces a new notion of versioned receivers so I wonder if longer-term we should think about adding rich support for that to the integration tests, e.g. having `enable_v1` alongside `enable_v2`, etc., so that we don't have to keep the folders separate.

Reply:
Unfortunately the folder name **is** coupled to the app name because of our feature that auto-detects which tests to run for a given PR.

I'm still thinking about what the right approach here is...

Reply:
Huh, I was able to get it to run just fine (I just had to add it to `supported_applications.txt`). I wasn't testing it with `-short` though.

Reply:
kokoro uses -short.

Reply:
@Mrod1598's [comment](https://github.com/GoogleCloudPlatform/ops-agent/pull/534#discussion_r859108740) on the IIS PR made me realize I wrongly assumed this was only testing one version -- in fact it's testing both simultaneously since it's named `mssql_v2`, so that might be OK for now. The downside still is that the two versions are not being tested in isolation.

Reply:
@jefferbrecht right. It was meant to be testing both at once. I'm glad you called it out though, as I forgot to add the `receiver_version` to this config.

Reply:
Talked offline with @martijnvans; we're OK with this approach but feel that a comment is warranted inside `enable` which clearly explains what's happening, e.g.

```
# This config gets merged with the built-in Ops Agent config, which already
# includes the v1 receiver in the default pipeline.
# Because the v2 receiver here uses a unique ID, it does not overwrite
# the default receiver and so both receivers will be active simultaneously
# during the test.
Add-Content (...)
```

CC @Mrod1598 since the same will apply to the IIS PR

Reply:
@jefferbrecht done